coopertunes/hparams/hparams.py

```diff
@@ -71,3 +72,6 @@ def _repr_object(self, v):
 
     def __repr__(self) -> str:
         return self._dumps()
+    
+    def get(self,name):
+        return vars(self)[name][0]
```

Comment:
<3

Reply:
YEEEE ITS NEEDED FOR ALL OF US TO FORGOR IT FASTTTTTTT

Some backstory: I acidentally saved myself one element tuples in Hparams attributes, and debbuged it for a while why tf it's automatically changing attributes to tuples (is it some weird feature of hparam?). It's a relict of a very frustrating half an hour